### ISSUE TITLE
Grid not HiDPI compatible

### ISSUE THREAD

**Issue by MarkJeronimus** [Jezzamonn/fourier#6]
`img-component-container` is not HiDPI compatible.

Normally:
[Imgur](https://i.imgur.com/MriXBVW.png)

HiDPI (can be emulated by scaling the browser to 200% zoom):

[Imgur](https://i.imgur.com/dJ6lr2s.png)

The tiles get more than twice as wide. The vertical pitch gets much less than 2 times as big. Text overlaps the images.

**Comment by MarkJeronimus** [comment]
Fixed image links (hopefully):

![Imgur](https://i.imgur.com/MriXBVW.png)

![Imgur](https://i.imgur.com/dJ6lr2s.png)

**Comment by Jezzamonn** [comment]
Interesting -- can you give some details of what browser you're running into this on? On my HDPI mac, it works fine on both Chrome and Safari

**Comment by Jezzamonn** [comment]
Chrome on hdpi mac at 100%:
<img width="913" alt="Screen Shot 2019-04-13 at 15 58 45" src="https://user-images.githubusercontent.com/10319630/56086138-10ba4100-5e05-11e9-9418-5fbf830e69d7.png">
Chrome on hdpi mac at 200%:
<img width="921" alt="Screen Shot 2019-04-13 at 15 58 34" src="https://user-images.githubusercontent.com/10319630/56086140-144dc800-5e05-11e9-9b20-ea96a1627f28.png">

**Comment by MarkJeronimus** [comment]
I see. It's an issue with browser compatibility.
Chrome-OK
Firefox-OK
Firefox ESR-Bad.

**Comment by Jezzamonn** [comment]
Right, it must be related to an older version of Firefox. Looking at the
"Can I use" it seems like versions 52-53 of firefox have some CSS Grid
related bugs -- maybe it's related to that?
https://caniuse.com/#feat=css-grid

It seems like the latest version of Firefox ESR should have the working
version of the grid, but I could be wrong.

**Comment by MarkJeronimus** [comment]
Yup, that's it. Sorry for accusing you of a bug.

**Comment by Jezzamonn** [comment]
No, no problem at all! Thanks for taking the time to flag it!

I suppose I could find a workaround but it might be more effort than
worth... :/
